### ISSUE TITLE
Linux: Cannot use keyboard navigation/actions when Korean Hangul input is active, cannot use Space when Japanese IME is active

### ISSUE THREAD

**Issue by hsandt** [renpy/renpy#4784]
OS: Linux Ubuntu 22.04 with Unity Desktop
Input system: standard Korean install, Mozc for Japanese

Note: looks like my input method was frozen, I have to restart it but now I cannot input Hangul nor Japanese kana anymore. So I'll have to wait and restart my PC to test once more if I can repro.

When Korean Hangul input is active, keyboard doesn't work in Renpy game for navigation and actions (text input works e.g. for Save page name, though).

When Japanese kana input is active, Space doesn't seem to advance text. If you mapped Space to button_select, it doesn't seem to work either.
